docs/src/_getting-started/meltano-at-a-glance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Meltano at a Glance
+description: If you want to know why you should use Meltano, here's a quick overview of the basics of all of it.
+layout: doc
+weight: 1
+---
+
+This document is intended to give you enough technical understanding of Meltano to becoming excited about it and wanting to use it! It's not going to teach you how to use it, we got Tutorials & How Tos for that. When you're ready to start your first Meltano project, we recommend you dive right into our Tutorial.
+
+### What users say
+
+_"For us it's a better day at work when we can use Meltano."_ - Nino Müller, Head of Technology at Substring
+
+
+_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as the data built tool (dbt)"._  - Martin Morset
+
+## Meltano at a glance
+Welcome to your Open Source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
+
+
+### Why companies love to build with Meltano
+
+- **No lock-in**: It's open source and has a strong community, you'll always be free.. (derisk)
+- **It's extensible from day 1**: It's super easy to add a custom connection using the SDKs/EDKs.
+- **Amazing developer experience**: Developers go from start to finish on new data projects, including extraction, loading, transforming & orchestrating data within days.
+- **Small surface area**: Features like "inline transformations" remove unnecessary steps from your data pipelines, and help companies stay compliant to security & GDPR regulations.
+
+### Key Features of Meltano, developers will love
+- **Start simple**: Meltano is pip-installable and comes in a prepackaged docker container, you can have your first ELT pipeline running litterally within minutes.
+- **DataOps out-of-the-box**: Meltano provides tools that make DataOps best practices easy to use in every project.
+- **Integrates with everything**: 300+ natively supported data sources & targets, as well as additional plugins like great expectations or dbt are natively available. 
+- **Easily customizable**: Meltano isn't just extensible, it's built to be extended! SDK & EDK are easy to use, plugins created by others are available on the Hub.
```

Comment:
```suggestion
- **Easily customizable**: Meltano isn't just extensible, it's built to be extended! The SDK for Singer Connectors & EDK for Meltano Components are easy to use. Meltano Hub helps you find all of the connectors and components created across the data community.
```

We haven't had to use EDK in a sentence yet, nor has it been near SDK.

Reply:
Mh ok. I really want to drop the "Singer" in front of connector though, and write "EDK for other Meltano plugins". We really need to focus in on the language here as it only gets more confusing.